packages/compiler-dom/src/transforms/vOn.ts

```diff
@@ -1 +1,26 @@
-// TODO
+import {
+  transformOn as baseTransform,
+  DirectiveTransform,
+  createObjectProperty,
+  createCallExpression
+} from '@vue/compiler-core'
+import { V_ON_MODIFIERS_GUARD } from '../runtimeHelpers'
+
+export const transformOn: DirectiveTransform = (dir, node, context) => {
+  const { modifiers } = dir
+  let baseResult = baseTransform(dir, node, context)
```

Comment:
```suggestion
  const baseResult = baseTransform(dir, node, context)
```

Reply:
Maybe we can set up [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint) and use it's `prefer-const` rule?